src/MusicalScore/Graphical/VexFlow/VexFlowMusicSheetCalculator.ts

```diff
@@ -295,152 +295,216 @@ export class VexFlowMusicSheetCalculator extends MusicSheetCalculator {
     return minStaffEntriesWidth;
   }
 
-  public calculateMeasureWidthFromLyrics(measuresVertical: GraphicalMeasure[], oldMinimumStaffEntriesWidth: number): number {
-    let elongationFactorForMeasureWidth: number = 1;
+  private calculateElongationFactor(containers: any[], staffEntry: GraphicalStaffEntry, lastEntryDict: any,
+                                    oldMinimumStaffEntriesWidth: number, elongationFactorForMeasureWidth: number,
+                                    measureNumber: number, oldMinSpacing: number, nextMeasureOverlap: number): number {
```

Comment:
I really like that there's one method for both lyrics and chords measure elongation now!

Reply:
Yeah, that's something that was bugging me too!  That whole DRY thing, right?